doc/nrf/device_guides/nrf53/qspi_xip_guide_nrf5340.rst

```diff
@@ -1,7 +1,8 @@
 .. _qspi_xip:
+.. _ug_nrf5340_intro_xip:
 
-Configuring external XIP
-########################
+Execute in place (XIP) configuration on the nRF5340 SoC
```

Comment:
I would keep 'externa' word somehow.
Rationale is that execution code form internal flash is also executing in place operation.